nvflare/private/fed/client/client_runner.py

```diff
@@ -193,10 +204,21 @@ def _process_task(self, task: TaskAssignment, fl_ctx: FLContext) -> Shareable:
         if filter_list:
             task_data = task.data
             for f in filter_list:
+                filter_name = f.__class__.__name__
                 try:
                     task_data = f.process(task_data, fl_ctx)
+                except UnsafeJobError:
+                    self.log_exception(fl_ctx, f"UnsafeJobError from Task Data Filter {filter_name}")
```

Comment:
I think it might be good to have the reason print out:

```
except UnsafeJobError as e:
    self.log_exception(fl_ctx, f"UnsafeJobError ({secure_format_exception(e)}) from Task Data Filter {filter_name}")
```

Reply:
Good idea!

Reply:
Implemented.